doc/rados/api/python.rst

```diff
@@ -20,8 +20,9 @@ perform object operations as a ``client.admin`` user.
 .. note:: To use the Ceph Python bindings, you must have access to a
    running Ceph cluster. To set one up quickly, see `Getting Started`_.
 
-First, create a Python source file for your Ceph client. ::
-   :linenos:
+First, create a Python source file for your Ceph client.
+
+.. prompt:: bash #
```

Comment:
```suggestion
.. prompt:: bash
```

since we already have `sudo` in the command line, there is no need to specify the `#` in the prompt. as it implies that the command requires the root privilege. 

the prompt defaults to `$` if `bash` shell is used. see https://github.com/sbrunner/sphinx-prompt#prompts

Reply:
I had not considered this, thank you.

What would you think about removing the `sudo` altogether? It doesn't sound like a good practice to edit a source file as root, and why would root permissions even be needed for editing a source file?

Reply:
thank you for thinking further down the road. yeah, neither does it make sense to me. no need to edit a source file as a root even if that file needs root permissions to run. and i doubt it in this case, the python binding does not require it, unless the default `ceph.conf` is only accessible by root.

agreed. we should drop the "sudo" from the command line.

Reply:
Updated PR title, PR body and commit short+long descriptions force pushed.